components/src/components/molecules/Textarea/Textarea.tsx

```diff
@@ -6,11 +6,13 @@ import { FieldBaseProps } from '../../atoms/Field'
 
 const TextArea = styled.textarea<{ $error?: boolean }>(
   ({ theme, disabled, $error }) => css`
-    background-color: ${theme.colors.transparent};
-    border-color: ${theme.colors.foregroundSecondary};
+    background-color: ${theme.colors.backgroundSecondary};
+    border-color: ${theme.colors.transparent};
     border-radius: ${theme.radii['2xLarge']};
-    border-width: ${theme.space['0.5']};
+    border-width: 1px;
```

Comment:
use px theme token

Reply:
Set the border to be the same as input...  '0.75'